sprites.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 
-#Copyright (c) 2007-8, Playful Invention Company.
-#Copyright (c) 2008-11 Walter Bender
+#Copyright (C) 2007-8, Playful Invention Company.
+#Copyright (C) 2008-11 Walter Bender
```

Comment:
would look more consistent with the other files if a space was placed after the "#" character

Reply:
sprites.py generates a PyGIWarning about PangoCairo; you didn't test?